Rohit/HTML basics remake/form.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>form</title>
+</head>
+<body>
+    <h1>Register</h1>
+ <form> 
+    <label for="First name">First name:</label> 
+    <input name="first name" type="text" placeholder="First name" required />
+    <label for="last name">Last name:</label> 
+    <input name="last name" type="text" placeholder="Last name" required /><br>
+    <br>
+    <label for="gender">gender:</label> 
+    <input  name="gender" type="radio" value="male" checked /> Male
+    <input  name="gender" type="radio" value="female" /> Female
+    <input  name="gender" type="radio" value="others" /> Others <br>
```

Comment:
Missing `required` attribute.

Reply:
@rohitraju1966 This still needs fixing. And male shouldn't be checked by default as it can end up in accidental submission and also offensive to other genders in certain cases.